client/src/services/fetch.js

```diff
@@ -0,0 +1,40 @@
+// GET
+export const fetchTodo = async () => {
```

Comment:
I believe the name should mention that this is a "list of TODOs"

Reply:
changed it